opentok/opentok.py

```diff
@@ -275,10 +276,10 @@ def headers(self):
         """For internal use."""
         return {
             'User-Agent': 'OpenTok-Python-SDK/' + __version__ + ' ' + platform.python_version(),
-            'X-TB-OPENTOK-AUTH': self._create_jwt_auth_header()
+            'X-OPENTOK-AUTH': self._create_jwt_auth_header()
         }
 
-    def archive_headers(self):
+    def api_headers(self):
```

Comment:
I think you're breaking the flow for archiving.

response = requests.post(self.archive_url(archive_id) + '/stop', headers=self.archive_headers(), proxies=self.proxies)

AttributeError: 'OpenTok' object has no attribute 'archive_headers'